tea/tea_test.go

```diff
@@ -0,0 +1,170 @@
+package tea_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/madlambda/crypt"
+	"github.com/madlambda/crypt/tea"
+	"github.com/madlambda/spells/assert"
```

Comment:
This is used for what?

Reply:
To assert stuff on tests, like comparing strings and checking errors